src/Core/Grid/Definition/Factory/OrderGridDefinitionFactory.php

```diff
@@ -0,0 +1,413 @@
+                ->setName($this->trans('Reference', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'reference',
+                ])
+            )
+            ->add((new BooleanColumn('new'))
+                ->setName($this->trans('New client', [], 'Admin.Orderscustomers.Feature'))
+                ->setOptions([
+                    'field' => 'new',
+                    'true_name' => $this->trans('Yes', [], 'Admin.Global'),
+                    'false_name' => $this->trans('No', [], 'Admin.Global'),
+                ])
+            )
+            ->add((new DataColumn('customer'))
+                ->setName($this->trans('Customer', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'customer',
+                ])
+            )
+            ->add((new OrderPriceColumn('total_paid_tax_incl'))
+                ->setName($this->trans('Total', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'total_paid_tax_incl',
+                    'is_paid_field' => 'paid',
+                ])
+            )
+            ->add((new DataColumn('payment'))
+                ->setName($this->trans('Payment', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'payment',
+                ])
+            )
+            ->add((new ColorColumn('osname'))
+                ->setName($this->trans('Status', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'osname',
+                    'color_field' => 'color',
+                ])
+            )
+            ->add((new DateTimeColumn('date_add'))
+                ->setName($this->trans('Date', [], 'Admin.Global'))
+                ->setOptions([
+                    'field' => 'date_add',
+                    'format' => $this->contextDateFormat,
+                ])
+            )
+            ->add((new LinkGroupColumn('pdf'))
+                ->setName($this->trans('PDF', [], 'Admin.Global'))
+                ->setOptions([
+                        'links' => [
+                            [
+                                'icon' => 'picture_as_pdf',
+                                'is_link_available_field' => 'is_invoice_available',
+                                'route' => 'admin_orders_generate_invoice_pdf',
+                                'route_param_name' => 'orderId',
+                                'route_param_field' => 'id_order',
+                            ],
+                            [
+                                'icon' => 'local_shipping',
+                                'is_link_available_field' => 'delivery_number',
+                                'route' => 'admin_orders_generate_delivery_slip_pdf',
+                                'route_param_name' => 'orderId',
+                                'route_param_field' => 'id_order',
+                            ],
+                        ],
```

Comment:
@matks for this config, do you think it's worth introducing objects here?

Reply:
For LinkGroupColumn options only ?

Reply:
Yeah, is it worth?

Reply:
No 😺